packages/wix-ui-tpa/src/components/NewCard/NewCard.st.css

```
@@ -1,3 +1,18 @@
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: applyUnit, fallback;
+}
+
+:vars {
+    /*Constance*/
+    DefalutRatio: 50;
+}
+
+:vars {
+    /*Overrides*/
+    Ratio: --overridable;
+}
```

Just a small small fix
```suggestion
/* Overrides */
:vars {
    /*  The ratio of a single card container */
    Ratio: --overridable;
}
```